### PR TITLE
Documentation: fixed CI badge

### PR DESCRIPTION
**Pull request by Alexander-Senko**
### Before

[Actions Status](https://github.com/drapergem/draper/actions?query=workflow%3Aci+branch%3Amaster)

### After

[Actions Status](https://github.com/drapergem/draper/actions?query=workflow%3Aci+branch%3Amaster)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Draper: View Models for Rails
 
-[![Actions Status](https://github.com/drapergem/draper/workflows/CI/badge.svg?branch=master)](https://github.com/drapergem/draper/actions?query=workflow%3Aci+branch%3Amaster)
+[![Actions Status](https://github.com/drapergem/draper/actions/workflows/ci.yml/badge.svg?branch=master )](https://github.com/drapergem/draper/actions?query=workflow%3Aci+branch%3Amaster)
```
Comment (y-yagi) [inline README.md:3]:
```suggestion
[![Actions Status](https://github.com/drapergem/draper/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/drapergem/draper/actions?query=workflow%3Aci+branch%3Amaster)
```

It seems that there is an unneeded space to the end.

**Review by y-yagi — APPROVED** [review]
LGTM!

**Review by y-yagi — APPROVED** [review]
👍

File: README.md (hunk 1 above)
Comment (Alexander-Senko) [reply README.md]: @y-yagi, you are really fast this time 😁 I noticed it too, but it took me some time to push the changes.

**Pull request merged**